packages/client/src/index.ts

```diff
@@ -20,53 +19,52 @@ export const extractBearerToken = (authorization: string): string => {
 };
 
 export const ensureBasicOptions = (options?: ConfigParameters): ConfigParameters => {
-  const { clientId, discoveryUrl, redirectUris } = options || {};
-  if (typeof discoveryUrl !== 'string' || !discoveryUrl.startsWith('http')) {
-    throw new Error('Invalid discoveryUrl');
+  const { clientId, logtoUrl } = options || {};
+  if (typeof logtoUrl !== 'string' || !logtoUrl.startsWith('http')) {
+    throw new Error('Invalid logtoUrl');
   }
 
   if (typeof clientId !== 'string' || clientId.length === 0) {
     throw new Error('Need clientId');
   }
 
-  if (redirectUris) {
-    if (!Array.isArray(redirectUris)) {
-      throw new TypeError('RedirectUris should be an array of strings');
-    }
-
-    for (const uri of redirectUris) {
-      if (typeof uri !== 'string') {
-        throw new TypeError('RedirectUris should be an array of strings');
-      }
-    }
-  }
-
   return {
     clientId,
-    discoveryUrl,
-    redirectUris,
+    logtoUrl,
   };
 };
 
+export const ensureTrailingSlash = (url: string): string => {
```

Comment:
should we keep a util or lib files for these methods?

Reply:
I raised this question in `logto`, can't find the link for now. The conclusion is to do it later.

Reply:
maybe "appendSlashIfNeeded" ? 

P.S. we can somehow follow [Swift API Design Guideline](https://swift.org/documentation/api-design-guidelines/) cc @simeng-li @xiaoyijun

Reply:
created [LOG-130](https://linear.app/silverhand/issue/LOG-130/standardize-dev-pipeline-for-silverhandessentials) for this